.github/workflows/stale-issues.yml

```
@@ -0,0 +1,57 @@
+        
+        # PR specific settings - We will not mark PRs as stale
+        days-before-pr-stale: -1  # Disable PR staling
+        days-before-pr-close: -1  # Disable PR closing
+        
+        # Exemptions
+        exempt-issue-labels: 'pinned,security,onhold,inprogress,Selected For Development,bug,enhancement,v3-alpha,high-priority'
+        exempt-all-issue-milestones: true
+        exempt-all-issue-assignees: true
+        
+        # Protection for existing issues
+        exempt-issue-created-before: '2024-01-01T00:00:00Z'
```

actionlint (1.7.7)

input "exempt-issue-created-before" is not defined in action "actions/stale@v9". available inputs are "any-of-issue-labels", "any-of-labels", "any-of-pr-labels", "ascending", "close-issue-label", "close-issue-message", "close-issue-reason", "close-pr-label", "close-pr-message", "days-before-close", "days-before-issue-close", "days-before-issue-stale", "days-before-pr-close", "days-before-pr-stale", "days-before-stale", "debug-only", "delete-branch", "enable-statistics", "exempt-all-assignees", "exempt-all-issue-assignees", "exempt-all-issue-milestones", "exempt-all-milestones", "exempt-all-pr-assignees", "exempt-all-pr-milestones", "exempt-assignees", "exempt-draft-pr", "exempt-issue-assignees", "exempt-issue-labels", "exempt-issue-milestones", "exempt-milestones", "exempt-pr-assignees", "exempt-pr-labels", "exempt-pr-milestones", "ignore-issue-updates", "ignore-pr-updates", "ignore-updates", "include-only-assigned", "labels-to-add-when-unstale", "labels-to-remove-when-stale", "labels-to-remove-when-unstale", "only-issue-labels", "only-labels", "only-pr-labels", "operations-per-run", "remove-issue-stale-when-updated", "remove-pr-stale-when-updated", "remove-stale-when-updated", "repo-token", "stale-issue-label", "stale-issue-message", "stale-pr-label", "stale-pr-message", "start-date"

(action)